magicctapipe/scripts/lst1_magic/magic_calib_to_dl1.py

```diff
@@ -319,23 +297,26 @@ def magic_calib_to_dl1(
                 )
 
             else:
+                # The UNIX format with the "long" sub-format can get the
+                # timestamp without affected by a rounding issue
```

Comment:
without being affected ... / without the rounding issue

Reply:
Thanks I changed it.